.github/workflows/ci.yml

```diff
@@ -148,3 +148,18 @@ jobs:
     needs: [e2e, bb-native-tests, bb-bench]
     steps:
       - run: echo Pull request merging now allowed.
+
+  notify:
+    needs: [e2e, bb-native-tests, bb-bench]
+    runs-on: ${{ github.actor }}-x86
```

Comment:
To be fair, this job can run pretty much anywhere. I'm using the same runner that's already up from other tests, but we could also just use `ubuntu-latest`. @ludamad thoughts?

Reply:
> this 'needs' the others, i thought this would just get cancelled if the others fail?

Not if there's a conditional defined, according to [github docs](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idneeds): _If a job fails or is skipped, all jobs that need it are skipped **unless the jobs use a conditional expression that causes the job to continue**._ (emph mine)